corehq/apps/case_search/models.py

```diff
@@ -244,6 +244,7 @@ class Meta(object):
         primary_key=True
     )
     enabled = models.BooleanField(blank=False, null=False, default=False)
+    synchronous_web_apps = models.BooleanField(blank=False, null=False, default=False)
```

Comment:
FYI making this non-nullable will lock the table during the migration, as it has to apply the default to all rows.  There are few enough `CaseSearchConfig`s that we might not need to worry about it, but wanted to flag that as a concern.

Reply:
Good to flag. I think we're fine here due to the size of the table (it's even smaller than I was expecting: ~250 rows).